### DISCUSSION TITLE
What files from NuGet packages are displayed in Visual Studio?

### DISCUSSION THREAD

**Discussion by paulomorgado** [NuGet/Home#13260] [Q&A]
Expanding a NuGet package in Visual Studio 2022 Solution Explorer shows several files from packages:

![image](https://github.com/NuGet/Home/assets/470455/5d040e51-b65a-4407-8aa6-54eeb3ff1d15)

What are the files shown?

Can additional files or file types be added?

**Comment by zivkan** [comment]
The https://github.com/dotnet/project-system team own this functionality. Although the code is in the NuGet.Client repo because of VS assembly load and plugin architecture issues.

**Reply by paulomorgado** [reply]
Thanks, @zivkan.

https://github.com/dotnet/project-system/discussions/9420
